plugins/synaptics-rmi/fu-synaptics-rmi-v7-device.c

```diff
@@ -443,9 +618,13 @@ fu_synaptics_rmi_v7_device_write_firmware(FuDevice *device,
 	fu_progress_set_id(progress, G_STRLOC);
 	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED);
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_BUSY, 0, "disable-sleep");
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_READ, 1, "verify-signature");
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 2, "fixed-location-data");
+	if (flash->bootloader_id[1] > 8)
+		fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 0, "flash-config");
```

Comment:
is 1100ms of delay really 0% of the total time?

Reply:
I'm not sure how to set this number because it's depending on the partition size or existence, the number even changes a little bit when I update the same firmware on same device.
But you are right, 1100ms is definitely not 0%, I can change that.